### PR TITLE
Adds a new felinid mutation

### PR DESCRIPTION
**Pull request by Boopideedoo**
## About The Pull Request
This pr adds a new mutation for felinids to have fun with
### Mapping March
Ckey to recieve rewards: N/A

## Why It's Good For The Game
This adds something to make up for the lack of mutations the feline species has compared to every other species. The first reason its good is it makes felinds have something that the others dont have for the first time.
Second reason, For too long have Felinids not had anything for fun but now they will atleast have something even if its not as great as hulk.
## Changelog
:cl:
add:uwuspeech mutation for felinids
/:cl:

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
 	"version": "0.2.0",
 	"configurations": [
+
```
Comment (ZephyrTFA) [inline .vscode/launch.json:4]: what happened here?

File: code/datums/mutations/speech.dm [hunk 2]
```diff
@@ -263,3 +263,31 @@
...
+	name = "uwuspeech"
+	desc = "A common genetic experiment gone wrong on felinds (Oh the horror kill whoever found this mutation)"
+	quality = POSITIVE
+	locked = TRUE
+	difficulty = 8
+	text_gain_indication = span_notice("<span class='notice'>You feel more uwu.</span>")
+	species_allowed = list(SPECIES_FELINE) //no talking this way for other species
+	text_lose_indication = "<span class='notice'>The feeling of uwu fades.</span>"
+/datum/mutation/human/uwuspeech/on_acquiring(mob/living/carbon/human/owner)
+	if(..())
```
Comment (ZephyrTFA) [inline code/datums/mutations/speech.dm:277]: Make these
```
. = ..()
if(.)
  return .
```

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]

**Comment by Boopideedoo** [comment]
> image
> 
> via https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#banned-content
> 
> I do not believe you have received any approval prior to opening this PR. I will be closing it until someone else is okay with it (specifically @optimumtact)

I didnt know it was supposed to be approved my bad

**Comment by JohnFulpWillard** [comment]
I don't think genetic mutations fall under that category, however this was attempted in https://github.com/tgstation/tgstation/pull/59802 and was denied

**Comment by san7890** [comment]
>Pull requests attempting to add or remove features from said races/species/mutants require prior approval as well.

Adding a felinid-only quirk _definitely_ falls under this category.

**Comment by ZephyrTFA** [comment]
I'm gonna go ahead and just give a hard no on this.

**Comment by Sealed101** [comment]
local coder asks in coderbus whether uwuspeech would be a good idea, gets several notices including from oranges that it would definitely not, PRs it anyway

real human bean

**Pull request closed without merging**